### DISCUSSION TITLE
Typical Banned Items

### DISCUSSION THREAD

**Discussion by bippity** [Pryaxis/TShock#2359] [General]
Reposting from the forums before it goes boom (https://tshock.co/xf/index.php?threads/typical-banned-items.2810/)

(Last Updated: December 13, 2016)
# Tile Editing

*   Bomb / Sticky Bomb / Bouncy Bomb / Fish Bomb(Bomb Fish) / Dynamite / Sticky Dynamite / Bouncy Dynamite / Explosives
*   Clentaminator
*   Dark Blue Solution / Purple Solution / Red Solution / Green Solution / Blue Solution
*   Unholy/holy / Blood water
*   Purification / Vile / Vicious Powder
*   Corrupt Seeds/Hallowed Seeds/Grass Seeds/Jungle Grass Seeds/Mushroom Grass Seeds/Crimson Seeds
*   Drill Containment Unit (TShock banned by default. Enabled through config file)

### Projectiles(use /projban)

    *   Drill Crosshair/DCU (453)
    *   Rocket II Grenade (136)
    *   Rocket II (137 & 339)
    *   Rocket II Proximity Mine (138)
    *   Rocket IV Grenade (142)
    *   Rocket IV (143 & 341)
    *   Rocket IV Proximity Mine (144)
    *   Pure Spray (145)
    *   Hallow Spray (146)
    *   Corrupt Spray (147)
    *   Mushroom Spray (148)
    *   Crimson Spray (149)

# Spam/Lag Prevention

*   1 Second Timer/3 Second Timer/5 Second Timer
*   Announcement Box

# Flooding/Exploit Prevention

*   Lava/Honey/Water bucket
*   Bottomless Water bucket
*   Super Absorbent Sponge
*   Inlet/Outlet Pump
*   Rod of Discord (TShock banned by default. Requires permission: _tshock.tp.rod_ )

# [List of Unused/Unobtainable items](http://terraria.gamepedia.com/Unobtainable):  

*   Red Potion
*   Solar Flare Axe / Chainsaw / Drill / Hammer
*   Nebula Axe / Chainsaw / Drill / Hammer
*   Vortex Axe / Chainsaw / Drill / Hammer
*   Stardust Axe / Chainsaw / Drill / Hammer

# Bugged/Glitched (With explanation)  

*   Optic Staff (Could cause FPS lag)

**Comment by bartico6** [comment]
Red Potion is no longer fully unobtainable: It can be legitimately obtained in For The Worthy worlds as chest loot.

**Comment by moisterrific** [comment]
There's a lot more projectiles that can enable easy griefing since they added a bunch of new liquid rocket types in 1.4.
